pkg/storage/article_storage.go

```diff
@@ -0,0 +1,19 @@
+package storage
+
+import (
+	"news-aggregator/pkg/model"
+	"time"
+)
+
+//go:generate mockgen -destination=mocks/mock_article.go -package=mocks news-aggregator/pkg/storage ArticleStorage
+
+// ArticleStorage is an interface that defines the methods for interacting with the article storage.
+type ArticleStorage interface {
```

Comment:
why article storage? try to be more abstract if so you can reuse it for different purposes

Reply:
This storage interface developed only for work with specified Article models for news.